src/util/selinux.c

```diff
@@ -326,6 +327,27 @@ int bus_selinux_check_send(BusSELinuxRegistry *registry,
         return 0;
 }
 
+static int bus_selinux_log(int type, const char *fmt, ...) {
+        _c_cleanup_(c_freep) char *message = NULL;
+        va_list ap;
+        int r;
+
+        va_start(ap, fmt);
+        r = vasprintf(&message, fmt, ap);
+        va_end(ap);
+        if (r < 0)
+                return r;
```

Comment:
What happens if we return `<0` from log-callbacks?

Reply:
I followed the behavior of the default log callback in libselinux, which returns negative error codes on error. However, looking through libselinux, it appears nothing catches these return values.